extensions/positron-r/package.json

```diff
@@ -87,6 +87,12 @@
         "title": "%r.command.packageTest.title%",
         "shortTitle": "%r.menu.packageTest.title%"
       },
+      {
+        "command": "r.cliHyperlinks",
+        "category": "R",
+        "title": "%r.command.cliHyperlinks.title%",
+        "shortTitle": "%r.menu.cliHyperlinks.title%"
+      },
```

Comment:
Will go away. Purely a development convenience, before I modified cli the way I needed.